resqpy/well/blocked_well.py

```diff
@@ -0,0 +1,3066 @@
+"""blocked_well.py: resqpy well module providing blocked well class.
```

Comment:
I would suggest making each of these "inner" submodules private (i.e. with a single underscore prefix), to avoid exposing them as a public api. That is, `_blocked_well.py` rather than `blocked_well.py`.

This will maintain the existing api: `from resqpy.well import BlockedWell` , but not expose the alternative `from resqpy.well._blocked_well import BlockedWell`. 

In turn, this keeps us free to refactor and reshuffle things internally without it being a breaking change.

Reply:
Thanks for the feedback! I've made the submodules private and fixed the init file. @andy-beer, I was wondering if I should makde the functions in resqpy\well\well_utils public also. These functions are only used by the well classes, and some have a "meta private" note in their docstring. Currently, these aren't listed in the init file.